### ISSUE TITLE
pylint docparams extension warns of missing params or missing types only if there is at least a param documented in rst docstrings

### ISSUE THREAD

**Issue by andrei-pavel** [pylint-dev/pylint#10138]
### Bug description
pylint docparams extension warns of missing params or missing types only if there is at least a param documented in rst docstrings.

This input results in no warnings:

```python
"""Module."""

def foonction(a, b):
    """Foo."""
    print(a, b)
```

This input:

```py
"""Module."""

def foonction(a, b):
    """Foo.

    :param a:
    """
    print(a, b)
```

Results in two warnings, which are expected, and desired:

```
a.py:3:0: W9015: "b" missing in parameter documentation (missing-param-doc)
a.py:3:0: W9016: "a, b" missing in parameter type documentation (missing-type-doc)
```

I wish there was a way to make it check for missing params even when no param is documented.
### Command used

```shell
pylint --load-plugins=pylint.extensions.docparams a.py
```

### Pylint output

```python
--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

### Expected behavior

```
a.py:3:0: W9015: "a, b" missing in parameter documentation (missing-param-doc)
a.py:3:0: W9016: "a, b" missing in parameter type documentation (missing-type-doc)
```

### Pylint version

```shell
pylint 3.3.1
astroid 3.3.6
Python 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910]
```

### OS / Environment

EndeavourOS 2024.09.22

**Comment by zenlyj** [comment]
thanks for the suggestion, would [missing-any-param-doc](https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/missing-any-param-doc.html) fit your use case?

**Comment by andrei-pavel** [comment]
Yes. It's what I was looking for. I searched in the docs, but not thorough enough.

**Comment by andrei-pavel** [comment]
Thank you, @zenlyj!
